cmfive.php

```diff
@@ -182,7 +278,11 @@ function installCoreLibraries()
     // dist     : []
     // names    : 2pisoftware/cmfive-core
 
-    $composer_json = sketchComposerForCore();
+    if (empty($branch)) {
```

Comment:
On my last message, the default value should be "null" since this function expects the ```$branch``` to always be set but checks here if it's empty. This ```empty``` call _should_ be ```is_null```, since ```empty```s ```isset``` check is redundant, much of a muchness really it can stay as empty.

Reply:
Dropped expectation of branch being filled, made this simply an advisory message.